open_instruct/rejection_sampling_multiple_rewards.py

```diff
@@ -0,0 +1,321 @@
+import os
+import numpy as np
+
+os.environ["HF_HUB_ENABLE_HF_TRANSFER"] = "1"
+
+import time
+import torch
+import torch.multiprocessing as mp
+from collections import defaultdict
+from dataclasses import dataclass
+from typing import Optional, Tuple, List
+from transformers import (
+    HfArgumentParser,
+    AutoModelForSequenceClassification,
+    DataCollatorWithPadding,
+    AutoTokenizer,
+)
+
+from tqdm import tqdm
+from datasets import Dataset
+import json
+from torch.utils.data import DataLoader
+from huggingface_hub import HfApi
+from collections import Counter
+from typing import Union
+
+api = HfApi()
+
+
+@dataclass
+class Args:
+    model_names_or_paths: tuple[str] = ("allenai/llama-3-tulu-2-8b-uf-mean-rm", "cleanrl/EleutherAI_pythia-1b-deduped__reward__tldr", "weqweasdas/RM-Mistral-7B")
```

Comment:
```suggestion
    model_names_or_paths: List[str] = field(default_factory=lambda: ["allenai/llama-3-tulu-2-8b-uf-mean-rm", "cleanrl/EleutherAI_pythia-1b-deduped__reward__tldr", "weqweasdas/RM-Mistral-7B"])
```

This enables us to tune `model_names_or_paths` in the command line :)

Reply:
Wonderful, thanks:)

Reply:
Fair enough, I just tested it with 4 RMs and it works fine so far :) 
Thanks, will merge and add documentation